packages/ra-ui-materialui/src/list/SingleFieldList.tsx

```diff
@@ -162,8 +162,6 @@ const Root = styled('div', {
 })(({ theme }) => ({
     display: 'flex',
     flexWrap: 'wrap',
-    marginTop: theme.spacing(-1),
```

Comment:
This looked bad in a SimpleShowLayout. It was designed to make Datagrid rows thinner, but we can do it in user land via the `sx` prop (see modified PostList).